prisma/schema.prisma

```diff
@@ -11,16 +11,18 @@ datasource db {
 }
 
 model HealthCareFacility {
-  uuid   String  @id @default(uuid())
-  shifts Shift[]
-  name   String
+  uuid           String          @id @default(uuid())
+  shifts         Shift[]
+  name           String
+  blockedWorkers BlockedWorker[]
```

Comment:
Blocked workers mapping belongs to Shift and not to Facility

Reply:
Blocked workers are not tied to Shift because of the requirement that any worker who is applying for any future shifts in that facility will not be allowed to do so i.e they will not be blocked from the facility(that's my understanding based on that requirement). If we stored the shift that the worker is blocked, are we just blocking the worker for that shift, actually no because of this requirement.